packages/teleport/src/components/DatabaseList/DatabaseList.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Column,
+  SortHeaderCell,
+  Cell,
+  renderLabelCell,
+  TextCell,
+  SortTypes,
+} from 'design/DataTable';
+import Table from 'design/DataTable/Paged';
+import isMatch from 'design/utils/match';
+import { sortBy } from 'lodash';
+import { Database } from 'teleport/services/database';
+
+function DatabaseList(props: Props) {
+  const { databases = [], searchValue = '', pageSize = 100 } = props;
```

Comment:
hm, 100 seems a little too big a number before paging, i think we use 20 everywhere else